libsolidity/parsing/Parser.cpp

```diff
@@ -804,13 +824,32 @@ ASTPointer<ModifierDefinition> Parser::parseModifierDefinition()
 		parameters = createEmptyParameterList();
 
 	ASTPointer<OverrideSpecifier> overrides;
+	bool isVirtual = false;
+
+	while(true)
```

Comment:
```suggestion
	while (true)
```

Reply:
This should be added to ``check_style.sh`` I guess...